yb-voyager/src/srcdb/common.go

```diff
@@ -17,3 +22,69 @@ func getExportedRowCount(tablesMetadata map[string]*utils.TableProgressMetadata)
 	}
 	return exportedRowCount
 }
+
+// Invoked at the end of export schema for Oracle and MySQL to process files containing statments of the type `\i <filename>.sql`, merging them together.
+func processImportDirectives(fileName string) error {
+	if _, err := os.Stat(fileName); os.IsNotExist(err) {
+		return nil
+	}
+	// Create a temporary file after appending .tmp extension to the fileName.
+	tmpFileName := fileName + ".tmp"
+	tmpFile, err := os.Create(tmpFileName)
+	if err != nil {
+		return fmt.Errorf("create %q: %w", tmpFileName, err)
+	}
+	defer tmpFile.Close()
+	// Open the original file for reading.
+	file, err := os.Open(fileName)
+	if err != nil {
+		return fmt.Errorf("open %q: %w", fileName, err)
+	}
+	defer file.Close()
+	// Create a new scanner and read the file line by line.
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := strings.Trim(scanner.Text(), " ")
+		// Check if the line contains the import directive.
+		if strings.HasPrefix(line, "\\i ") {
+			// Split the line into tokens.
+			tokens := strings.Split(line, " ")
+			// Check if the line contains the correct number of tokens.
+			if len(tokens) != 2 {
+				return fmt.Errorf("invalid number of tokens in line: %s", line)
+			}
+			// Check if the file exists.
+			importFileName := strings.Trim(tokens[1], "'")
+			log.Infof("Inlining contents of %q in %q", importFileName, fileName)
+			if _, err = os.Stat(importFileName); err != nil {
```

Comment:
here you must be checking the existince of `importFileName` as `os.Open()` is used at next line.
Again you can use `utils.FileOrFolderExists()` here

Reply:
I would suggest to keep it as it is. :-) Let the code fail if the file is missing. We need to know why it is failing.